src/main/java/com/javachallengers/simpson/model/dto/ApiErrorDTO.java

```diff
@@ -3,35 +3,16 @@
 import java.io.Serializable;
 import java.time.LocalDateTime;
 
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
 public final class ApiErrorDTO implements Serializable {
 	private static final long serialVersionUID = 5160147982546412023L;
 
 	private final String error;
```

Comment:
Do we need to expose all our fields here?

Reply:
No, we don't. Changed from @Data to @Getter